screens/Cart/CheckoutScreen.tsx

```diff
@@ -1,16 +1,276 @@
+            placeholder="CVV"
+            maxLength={3}
+            keyboardType="numeric"
+            style={[styles.longInput, { width: 50, marginHorizontal: 5 }]}
+          />
+
+          <TextInput placeholder="Name on Card" style={styles.longInput} />
+        </View>
+
+        <Text style={[styles.subTitle, { paddingVertical: 10 }]}>
+          Final Review
+        </Text>
+        <View style={{ flexDirection: "row", justifyContent: "center" }}>
+          <View
+            style={{
+              borderRadius: 10,
+              backgroundColor: colors.white,
+              width: 350,
+              height: 300,
+              elevation: 10,
+              marginBottom: 20,
+              flexDirection: "row",
+              justifyContent: "space-between",
+            }}
+          >
+            <View
+              style={{
+                justifyContent: "space-around",
+                paddingLeft: 20,
+                width: 150,
+                flex: 1,
+              }}
+            >
+              <Text style={styles.orderTitle}>Subtotal:</Text>
+              <Text style={styles.orderTitle}>Shipping & Handling:</Text>
+              <Text style={styles.orderTitle}>Tax:</Text>
+              <Text style={styles.orderTitle}>Grand Total:</Text>
+            </View>
+
+            <View
+              style={{
+                justifyContent: "space-around",
+                paddingLeft: 20,
+                width: 150,
+                flex: 1,
+              }}
+            >
+              <Text style={styles.orderTitle}>${getTotal()}</Text>
+              <Text style={styles.orderTitle}>$5.99</Text>
+              <Text style={styles.orderTitle}>${totalTax.toFixed(2)}</Text>
+              <Text style={styles.orderTitle}>
+                ${getTotal() + totalTax + 5.99}
+              </Text>
+            </View>
+          </View>
+        </View>
+
+        <View style={styles.placeOrderBox}>
+          <TouchableOpacity style={styles.placeOrderButton}>
+            <Text style={{ color: colors.white }}>Place Order</Text>
+          </TouchableOpacity>
+        </View>
+      </ScrollView>
     </View>
   );
 };
 
 export default CheckoutScreen;
 
 const styles = StyleSheet.create({
-
+  title: {
+    fontSize: 25,
+    padding: 15,
+  },
+  subTitle: {
+    fontSize: 18,
+    paddingHorizontal: 15,
+  },
+  orderTitle: {
+    fontSize: 20,
+  },
+  titleBox: {
+    flexDirection: "row",
+    height: 75,
+    alignItems: "center",
+  },
+  placeOrderBox: { flexDirection: "row", justifyContent: "center" },
+  placeOrderButton: {
+    backgroundColor: colors.violet,
+    flexDirection: "row",
+    justifyContent: "center",
+    width: 200,
+    borderRadius: 10,
+    padding: 10,
+    elevation: 10,
+    marginBottom: 20,
+  },
+  longInput: {
+    backgroundColor: colors.white,
+    paddingHorizontal: 10,
+    width: 375,
+    marginVertical: 5,
+    borderRadius: 10,
+    elevation: 10,
+  },
 });
```

Comment:
I like the logic you implemented in here .I  think using and  creating one or more component will make rendering it here would make the code in this page shorter and neat as you can also  avoid the long inline styles lines you have in here .

Reply:
Noted!